src/com/company/Main.java

```diff
@@ -1,8 +1,270 @@
 package com.company;
 
+import java.io.IOException;
+import java.util.Scanner;
+
+import static com.company.Texts.*;
+
 public class Main {
 
-    public static void main(String[] args) {
-	// write your code here
+    public static final int MIN_BOARD_SIZE = 1;
+    public static final int MAX_BOARD_SIZE = 9;
+    private static final int MIN_BOARD_ID = 1;
+    private static final char BAD_SIGN = '@';
+    private static final char YES = 'T';
+    private static final int ItIsO = 2; //oznacza O w tabeli//
```

Comment:
`ItIsO`? ... To już lepiej byłoby zrobić samo `O`. Zamień to na `circle` (i `cross` poniżej) proszę.
A jak dojdziemy do lekcji o Enumach to to zmienimy.